php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -332,6 +332,13 @@ abstract class NDB_BVL_Instrument extends NDB_Page
     {
         $DB = $this->loris->getDatabaseConnection();
 
+        //check if user has site's permission
+        $sessionID = $this->getSessionID();
+        if ($sessionID !== null) {
+            $timepoint = \Timepoint::singleton($sessionID);
+            return $timepoint->isAccessibleBy($user);
```

Comment:
This is now bypassing the instrument permissions checks below when sessionID is set. It should only be returning here if *not* accessible. If the timepoint is accessible it still needs to do the instrument permission checks.